Magento_LayeredNavigation/templates/layer/state.phtml

```diff
@@ -14,10 +21,11 @@
                     <a
                         href="<?= $_filter->getRemoveUrl() ?>"
                         class="active-filters__remove"
-                        aria-label="<?= __('Disable filter') ?>"
-                        title="<?= __('Disable filter') ?>"
+                        aria-label="<?= __('Disable filter') . ': ' . $block->stripTags($_filter->getLabel()) ?>"
+                        title="<?= __('Disable filter') . ': ' . $block->stripTags($_filter->getLabel()) ?>"
                     >
-                        <svg class="icon active-filters__remove-icon">
+                        <svg class="icon active-filters__remove-icon" role="img">
+                            <title><?= __('Close') ?></title>
                             <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#close') ?>"></use>
                         </svg>
                     </a>
```

Comment:
Line 40 in this file - there is an `a` link inside the list as a child of `ul` which is forbidden. only li coan be the children of ul. Make a fix, please.

Reply:
`<li>` added

Reply:
👍